spec/models/spree/customer_return_spec.rb

```diff
@@ -0,0 +1,128 @@
+describe Spree::CustomerReturn, type: :model do
+
+  # => Actually, just in case you want to regenerate vcr you must manually
```

Comment:
There is a TODO annotation that describes how to generate orders in shipwire with particular state via shipwire API. Actually I don't know if this is possible but I've left the instructions for a future implementation. I could move these notes to a new issue. What do you think about @DanielePalombo ?